parameter-sets/warp10connection/parameter-set.json

```diff
@@ -10,15 +10,10 @@
 
     "params": [
         {
-            "name": "host",
-            "label": "Warp10 hostname",
+            "name": "base_url",
+            "label": "Warp10 base URL",
             "type": "STRING",
-            "mandatory": true
-        },
-        {
-            "name": "port",
-            "label": "Warp10 port",
-            "type": "INT",
+            "description": "Enter as: https://warp10host:warp10port",
```

Comment:
I think we have to cover the case of self generated ssl certificate. Was it a requirement? if not it could be in a v1.1

Reply:
Hmm, you are certainly right, I didn't think too much about it. I'm not sure if we should handle it now or in v1.1

Reply:
Addressed in 0a9bf27